src/app/pages/Map/Map.tsx

```diff
@@ -233,15 +233,17 @@ export default function MapBox(): JSX.Element {
     map.current && midpoint
       ? new mapboxgl.Marker().setLngLat(midpoint).addTo(map.current)
       : null
-
+  marker
```

Comment:
what's marker? this is new to me :D

Reply:
Hey Iva :) its the marker on the map, in the picture the light blue one. I constructed it `new mapboxgl.Marker()` told him where to be `.setLngLat(midpoint)` and put it to the map here `.addTo(map.current)`.  
This one marker here in line 236 is only there, because otehrwhise typescript throws an error and the build will fail. `the constante maker is not used`(or smth like this). I need to fine a better solution for this, but is works :)